### ISSUE TITLE
Get-DscConfiguration will throw when xWebsite returns $null LogCustomFields

### ISSUE THREAD

**Issue by regedit32** [PowerShell/xWebAdministration#372]
Get-DscConfiguration is very particular about the data type that is returned.  In the MOF String LogCustomFields is defined as an array, but if $website.logFile.customFields.Collection is $null, then $null is  being included in the Get return, which is not the same as an empty array.  $cimLogCustomFields should be casted as an array.

**Comment by johlju** [comment]
@regedit32 Do you have the error message that Get-DscConfiguration is throwing when this happen?

Looking for if we should add an example for testing the types in the unit and/or integration [test template](https://github.com/PowerShell/DscResources/tree/master/Tests.Template).

**Comment by regedit32** [comment]
Here's the error message:
```
Get-DscConfiguration : GetConfiguration did not succeed.
At line:1 char:1
+ Get-DscConfiguration
+ ~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (MSFT_DSCLocalConfigurationManager:root/Microsoft/...gurationManager) [Get-DscConfiguration], CimException
    + FullyQualifiedErrorId : MI RESULT 1,Get-DscConfiguration
```

**Comment by johlju** [comment]
@regedit32 submitted an issue https://github.com/PowerShell/DscResource.Tests/issues/239 to track adding tests to test framework or template for this.

**Comment by johlju** [comment]
Closing this as this was resolved in PR #373.
